addons/portal/static/src/js/portal.js

```diff
@@ -34,7 +34,7 @@ publicWidget.registry.portalDetails = publicWidget.Widget.extend({
         var countryID = ($country.val() || 0);
         this.$stateOptions.detach();
         var $displayedState = this.$stateOptions.filter('[data-country_id=' + countryID + ']');
-        var nb = $displayedState.appendTo(this.$state).show().length;
+        var nb = $displayedState.appendTo(this.$state).removeClass('d-none').show().length;
```

Comment:
Would it make sense to add a comment here explaining why we're adding `removeClass('d-none')`?